yarn.lock

```diff
@@ -1012,6 +1012,13 @@
   dependencies:
     regenerator-runtime "^0.13.2"
 
+"@babel/runtime@^7.4.0":
```

Comment:
Would you mind deleting all the `@babel/runtime` chunks adjacent to this one and then re-run `yarn kbn bootstrap` to coalesce the appropriate versions?